misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/TestProcessor.java

```diff
@@ -518,4 +528,27 @@ private void processTestAnnotation(AnnotationNode annotationNode, String functio
         }
     }
 
+    /**
+     * Get the field name string from {@code SpecificFieldNode}.
+     *
+     * @param specificField SpecificFieldNode
+     * @return String
+     */
+    private String getFieldName(SpecificFieldNode specificField) {
+        Node fieldNameNode = specificField.fieldName();
+
+        if (fieldNameNode.kind() != SyntaxKind.STRING_LITERAL) {
+            return ((Token) fieldNameNode).text();
+        }
+
+        String fieldName = ((BasicLiteralNode) fieldNameNode).literalToken().text();
+        if (fieldName.length() > 1 && fieldName.charAt(fieldName.length() - 1) == '"') {
+            fieldName = fieldName.substring(1, fieldName.length() - 1);
+        } else {
+            // Missing end quote case
```

Comment:
A missing end quote would give a compilation error. So, I think we can simply omit this else block. WDYT?

Reply:
+1